src/command_modules/azure-cli-network/azure/cli/command_modules/network/_params.py

```diff
@@ -332,7 +332,7 @@ def completer(prefix, action, parsed_args, **kwargs): # pylint: disable=unused-a
 register_cli_argument('network lb create', 'dns_name_type', help=argparse.SUPPRESS)
 register_cli_argument('network lb create', 'public_ip_address_allocation', choices=choices_ip_allocation_method, default='dynamic', type=str.lower)
 register_folded_cli_argument('network lb create', 'public_ip_address', 'Microsoft.Network/publicIPAddresses', validator=validate_public_ip_type)
-register_folded_cli_argument('network lb create', 'subnet', 'subnets', parent_name='virtual_network_name', parent_type='Microsoft.Network/virtualNetworks')
+register_folded_cli_argument('network lb create', 'subnet', 'subnets', parent_name='virtual_network_name', parent_type='Microsoft.Network/virtualNetworks', default_value_flag='none')
```

Comment:
>default_value_flag='none') [](start = 156, length = 26)

According to the azuredeploy.json, the only allowed values for virtual_network_type are "new", "existingId" and "existingName". This seems like it would result in the error "'none' is not a value in enum ..."? If None is not a viable option, you should have "none_flag_value=None".